build/config/PublishData.json

```diff
@@ -5,56 +5,28 @@
             "version": "2.8.*",
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-            "channels": [ "dev15.8" ]
-        },
-        "dev15.6-preview2-vs-deps": {
-            "nugetKind": "PerBuildPreRelease",
-            "version": "2.7.*",
-            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
-            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-            "channels": [ "dev15.6p2" ]
-        },
-        "dev15.6.x-vs-deps": {
-            "nugetKind": "PerBuildPreRelease",
-            "version": "2.7.*",
-            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
-            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-            "channels": [ "dev15.6p4", "dev15.6" ]
+            "channels": [ "dev16" ]
         },
         "dev15.7.x-vs-deps": {
             "nugetKind": "PerBuildPreRelease",
             "version": "2.8.*",
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-            "channels": [ "dev15.7p2", "dev15.7" ]
+            "channels": [ "dev15.7" ]
         },
-        "dev15.7-preview1-vs-deps": {
-            "nugetKind": "PerBuildPreRelease",
-            "version": "2.8.*",
-            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
-            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-            "channels": [ "dev15.7p1"]
-        },
-        "dev15.8-preview1-vs-deps": {
-            "nugetKind": "PerBuildPreRelease",
-            "version": "2.9.*",
-            "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
-            "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-            "channels": [ "dev15.8p1"]
-        },
-       "dev15.8-preview3-vs-deps": {
+        "dev15.8.x-vs-deps": {
             "nugetKind": "PerBuildPreRelease",
             "version": "2.9.*",
             "nuget": [ "https://dotnet.myget.org/F/roslyn/api/v2/package" ],
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
-            "channels": [ "dev15.8p3"]
+            "channels": [ "dev15.8p5"]
```

Comment:
Should also include dev15.8 too.

@jaredpar is anything still consuming this yet? This has been a commons source of confusion by our swat team of how to update this file.

Reply:
Yes. I commonly point people to our channel entries on dotnet/versions when they want to consume specific feeds of roslyn.

What exactly is the confusion here?